app/src/main/java/com/rokt/roktdemo/ui/layouts/ScanQrViewModel.kt

```diff
@@ -58,6 +58,7 @@ class ScanQrViewModel @Inject constructor(
             val attributes = hashMapOf(
                 ATTRIBUTE_IS_DEMO to true.toString(),
                 ATTRIBUTE_LAYOUT_ID to data.previewId,
+                ATTRIBUTE_VERSION_ID to data.versionId,
```

Comment:
versionId is already present?
Is this deployment backward compatible?

Reply:
Yes, `versionId` is already returned in the `/preview` response: https://github.com/ROKT/rokt-demo-android/blob/release-1.2.x/app/src/main/java/com/rokt/roktdemo/ui/layouts/model/PreviewData.kt#L6

Reply:
https://github.com/ROKT/ecommerce-api/blob/a060eadfa305304af272fbd97b922585bd31061f/mobile-api/src/Rokt.Widget.Presentation.Mobile.ApiControllers/Controllers/V1/ExperiencesV1Controller.cs#L93-L97